api/envoy/extensions/filters/http/ext_proc/v3/ext_proc.proto

```diff
@@ -205,6 +206,9 @@ message ExternalProcessor {
 
   // Options related to the sending and receiving of dynamic metadata.
   MetadataOptions metadata_options = 16;
+
+  // Optional retry policy for the external grpc stream.
```

Comment:
How would this work with ext_proc that needs to send more than 1 message? I.e. headers + body? If you retry on the body you can end up on a completely new service.

Reply:
nice catch! 
I am about to add another issue to track this, I think we need a mechanism in Envoy retry-policy impl to stop retrying when it's grpc and any response has been received. 

A similar behavior described in here: 
https://learn.microsoft.com/en-us/aspnet/core/grpc/retries?view=aspnetcore-8.0#streaming-calls

Reply:
Sounds good. However it seems like we need to first address the issue with retries of streaming requests before adding this policy.

Reply:
This feature is not enabled if not configured. 

The issue we discussed here(if true) is the same for Envoy that talks to a grpc server upstream with retry policy on. I agree with we need fix it if it's an issue with our retry policy impl. but probably a blocker for this PR?